app/views/items/_item.html.erb

```diff
@@ -0,0 +1,3 @@
+<% item.each do |item| %>
```

Comment:
Generally the partial name of `_item` would indicate this represents a single item